src/um/winbase.rs

```diff
@@ -361,6 +361,115 @@ pub const MS_CTS_ON: DWORD = 0x0010;
+    ProcThreadAttributeValue (ProcThreadAttributeMitigationPolicy, FALSE, TRUE, FALSE)
+#endif
+#if (_WIN32_WINNT >= _WIN32_WINNT_WIN8)
+#define PROC_THREAD_ATTRIBUTE_SECURITY_CAPABILITIES \
+    ProcThreadAttributeValue (ProcThreadAttributeSecurityCapabilities, FALSE, TRUE, FALSE)
+#endif
+#define PROC_THREAD_ATTRIBUTE_PROTECTION_LEVEL \
+    ProcThreadAttributeValue (ProcThreadAttributeProtectionLevel, FALSE, TRUE, FALSE)
+#if (_WIN32_WINNT >= _WIN32_WINNT_WINBLUE)
+#endif
+*/
+pub const PROC_THREAD_ATTRIBUTE_NUMBER: DWORD = 0x0000FFFF;
+pub const PROC_THREAD_ATTRIBUTE_THREAD: DWORD = 0x00010000;
+pub const PROC_THREAD_ATTRIBUTE_INPUT: DWORD = 0x00020000;
+pub const PROC_THREAD_ATTRIBUTE_ADDITIVE: DWORD = 0x00040000;
+pub const ProcThreadAttributeParentProcess: DWORD = 0;
+pub const ProcThreadAttributeHandleList: DWORD = 2;
+pub const ProcThreadAttributeGroupAffinity: DWORD = 3;
+pub const ProcThreadAttributePreferredNode: DWORD = 4;
+pub const ProcThreadAttributeIdealProcessor: DWORD = 5;
+pub const ProcThreadAttributeUmsThread: DWORD = 6;
+pub const ProcThreadAttributeMitigationPolicy: DWORD = 7;
+pub const ProcThreadAttributeSecurityCapabilities: DWORD = 9;
+pub const ProcThreadAttributeProtectionLevel: DWORD = 11;
+pub const ProcThreadAttributeJobList: DWORD = 13;
+pub const ProcThreadAttributeChildProcessPolicy: DWORD = 14;
+pub const ProcThreadAttributeAllApplicationPackagesPolicy: DWORD = 15;
+pub const ProcThreadAttributeWin32kFilter: DWORD = 16;
+pub const ProcThreadAttributeSafeOpenPromptOriginClaim: DWORD = 17;
+pub const ProcThreadAttributeDesktopAppPolicy: DWORD = 18;
+// TODO: constants for mitigation policy
+macro_rules! DEFINE_PROC_THREAD_ATTRIBUTE {
+    () => {};
+    ($name:ident = $value:ident: $thread:expr, $input:expr, $additive:expr; $($rest: tt)*) => {
+        pub const $name: DWORD = ($value & PROC_THREAD_ATTRIBUTE_NUMBER) | 
+          ($thread * PROC_THREAD_ATTRIBUTE_THREAD) |
+          ($input * PROC_THREAD_ATTRIBUTE_INPUT) |
+          ($additive * PROC_THREAD_ATTRIBUTE_ADDITIVE);
+        DEFINE_PROC_THREAD_ATTRIBUTE!($($rest)*);
+    };
+}
+DEFINE_PROC_THREAD_ATTRIBUTE! {
+    PROC_THREAD_ATTRIBUTE_PARENT_PROCESS = ProcThreadAttributeParentProcess: 0,1,0;
+    PROC_THREAD_ATTRIBUTE_HANDLE_LIST = ProcThreadAttributeHandleList: 0,1,0;
+    PROC_THREAD_ATTRIBUTE_GROUP_AFFINITY = ProcThreadAttributeGroupAffinity: 1,1,0;
+    PROC_THREAD_ATTRIBUTE_PREFERRED_NODE = ProcThreadAttributePreferredNode: 0,1,0;
+    PROC_THREAD_ATTRIBUTE_IDEAL_PROCESSOR = ProcThreadAttributeIdealProcessor: 1,1,0;
+    PROC_THREAD_ATTRIBUTE_UMS_THREAD = ProcThreadAttributeUmsThread: 1,1,0;
+    PROC_THREAD_ATTRIBUTE_MITIGATION_POLICY = ProcThreadAttributeMitigationPolicy: 0,1,0;
+    PROC_THREAD_ATTRIBUTE_SECURITY_CAPABILITIES = ProcThreadAttributeSecurityCapabilities: 0,1,0;
+    PROC_THREAD_ATTRIBUTE_PROTECTION_LEVEL = ProcThreadAttributeProtectionLevel: 0,1,0;
+}
+// TODO: more attributes
```

Comment:
I'm not sure if this TODO is needed, unless it was meant for yourself before merging?

Reply:
yes, this TODO is for me